require-semver-guidance-label/README.md

```diff
@@ -0,0 +1,89 @@
+# uwit-iam/actions/require-semver-guidance-label
+
+This action requires that a pull request have a 
+label in the format off `semver-guidance:foo`, where `foo`
+is either an explicit version (e.g., `1.2.3`) or a 
+guidance directive such as `prerelease`, `patch`, `minor`, `major`. 
+
+If you include this action in your pull request workflow, your validations will fail until 
+an authorized user (or some automated task) sets version guidance for a pull request as part of the change review process.
+
+The intended use is that you can require a pull request to have one of these 
+labels before it can be approved, so that your CICD workflow can automatically
+set the version for the change when it is merged into your default branch.
```

Comment:
Which is left as an exercise for the reader?

Reply:
Yes, or as an exercise in patience while we expand this collection of actions.